src/main/java/org/broadinstitute/hellbender/tools/spark/sv/StructuralVariationDiscoveryArgumentCollection.java

```diff
@@ -119,6 +119,10 @@
         @Argument(doc = "output dir for assemblies", fullName = "gfaDir", optional = true)
         public String gfaDir;
 
+        @Argument(doc = "file for non-assembled breakpoints in bedpe format",
```

Comment:
seems obvious to us, but you might want to say "output file"

Reply:
done